app/models/miq_worker.rb

```diff
@@ -434,10 +434,6 @@ def status_update
     rescue Errno::ESRCH
       update(:status => STATUS_ABORTED)
       _log.warn("No such process [#{friendly_name}] with PID=[#{pid}], aborting worker.")
-    rescue => err
-      # Calling ps on Linux with a pid that does not exist fails with a RuntimeError containing an empty message.
-      # We will ignore this since we may be asking for the status of a worker who has exited.
-      _log.warn("#{self.class.name}: #{err.message}, while requesting process info for [#{friendly_name}] with PID=[#{pid}]") unless err.message.blank?
```

Comment:
No, see the 3rd commit.  We should handle generic errors when they are a problem.  I'm not sure how to handle them since they shouldn't happen.  I guess we can blindly swallow them but I really think lots has changed since 2008 so it should raise the Errno exception or nothing